app/models/s3_csv_export.rb

```diff
@@ -18,7 +18,7 @@ def create
   private
 
   def s3_bucket
-    Aws::S3::Resource.new.bucket ENV['AWS_BUCKET']
+    Aws::S3::Resource.new.bucket Rails.application.secrets.aws_bucket
```

Comment:
I've been using the "Artsy data" bucket in `.env`, but now that I think about it we should make a practice to use the Bearden staging and production buckets. (Note to self.)